systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp

```diff
@@ -2,6 +2,7 @@
 #include "RigidBodyManipulator.h"
 #include "RigidBodyConstraint.h"
 #include "drakeUtil.h"
+#include "controlUtil.h"
```

Comment:
sounds like it should probably go into drakeUtil. 

On Apr 1, 2015, at 4:19 PM, Robin Deits <notifications@github.com> wrote:

> In systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp:
> 
> > @@ -2,6 +2,7 @@
> >  #include "RigidBodyManipulator.h"
> >  #include "RigidBodyConstraint.h"
> >  #include "drakeUtil.h"
> > +#include "controlUtil.h"
> when I did the controller reorg, I created a sizecheck() and put it in controlUtil. We can certainly move it elsewhere
> 
> —
> Reply to this email directly or view it on GitHub.
>